src/Microsoft.AspNetCore.Mvc.Core/Internal/DefaultAssemblyPartDiscoveryProvider.cs

```diff
@@ -26,7 +26,7 @@ public static class DefaultAssemblyPartDiscoveryProvider
             "microsoft.aspnetcore.mvc.formatters.xml",
             "microsoft.aspnetcore.mvc.localization",
             "microsoft.aspnetcore.mvc.razor",
-            "microsoft.aspnetcore.mvc.razor.host",
+            "microsoft.aspnetcore.mvc.razor.extensions",
```

Comment:
We should probably have `Razor.Runtime` here. There's not a whole lot of utility you could get only referencing `microsoft.aspnetcore.mvc.razor.extensions`. It was probably true with the host too.